state/state.go

```diff
@@ -257,6 +260,10 @@ func (self *State) Next() (err error) {
 	return
 }
 
+func (self *State) PreviouslyAppliedOrders() map[godip.Province]godip.Adjudicator{
```

Comment:
Please add a comment specifying exactly what this means, when it's empty, and when it's populated.

Reply:
Have a look at it if you find it sufficient. 😄